src/constants.py

```diff
@@ -0,0 +1,56 @@
+import os

```

Comment:
You may want in-line type annotations for your constants in this file so you can see their data types at a glance.